docs/good-practices/conventional-comments.md

```diff
@@ -0,0 +1,6 @@
+# Conventional comments
```

Comment:
**suggestion**: The second level menu items are in mixed conventions. It feels natural to change strings separated by hyphens to spaces.
![image](https://user-images.githubusercontent.com/13812896/203008358-08c036c4-bb25-4189-9a47-f78d1e8e7ab9.png)

Reply:
I'm gonna fix it 👍🏻

Reply:
I'm not able to reproduce 😞

Reply:
I'm not able to reproduce error 😞